owl/EASE-OBJ.owl

```diff
@@ -3470,9 +3526,29 @@ It is also possible that a Transient transitionsBack to the initial object. An e
     <!-- http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#SocialObject -->
 
     <rdf:Description rdf:about="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#SocialObject"/>
+    
+
+
+    <!-- 
+    ///////////////////////////////////////////////////////////////////////////////////////
+    //
+    // General axioms
+    //
+    ///////////////////////////////////////////////////////////////////////////////////////
+     -->
+
+    <rdf:Description>
+        <rdf:type rdf:resource="http://www.w3.org/2002/07/owl#AllDisjointClasses"/>
```

Comment:
Probably best to leave this out for now. It seems silly that Temperature, Material, GeometricalQuality, DeviceState are on the same level in the taxonomy

Reply:
I have removed Geometric class and put all sub-classes on same level as Material and Temperature, etc.